docs/source/nodes_and_pipelines/micro_packaging.md

```diff
@@ -73,13 +73,13 @@ You can pull a micro-package from a tar file by executing `kedro micropkg pull <
 * Kedro will also parse any requirements packaged with the micro-package and add them to project level `requirements.in`.
 * It is advised to do `kedro build-reqs` to compile the updated list of requirements after pulling a micro-package.
 
-```eval_rst
-.. note::  If a micro-package has embedded requirements and a project ``requirements.in`` file does not already exist, it will be generated based on the project ``requirements.txt`` before appending the micro-package requirements.
+```{note}
+If a micro-package has embedded requirements and a project ``requirements.in`` file does not already exist, it will be generated based on the project ``requirements.txt`` before appending the micro-package requirements.
```

Comment:
```suggestion
If a micro-package has embedded requirements and a project `requirements.in` file does not already exist, it will be generated based on the project `requirements.txt` before appending the micro-package requirements.
```